src/main/java/org/example/Cinema.java

```diff
@@ -0,0 +1,74 @@
+package org.example;
+
+public class Cinema {
+    private int row = 5;
+    private int place = 10;
+    private int[][] cinema = new int[row][place];
+
+    public Cinema() {
+        for(int i = 0; i < cinema.length; i++){
+            for (int j = 0; j < cinema[i].length; j++){
+                cinema[i][j] = j + 1;
+            }
+        }
+    }
+
+    public Cinema(int[][] cinema) {
+        this.cinema = cinema;
+    }
+
+    public int[][] getCinema() {
+        return cinema;
+    }
+
+    public void setCinema(int[][] cinema) {
+        this.cinema = cinema;
+    }
+
+    public void printCinema() {
+        for (int i = 0; i < cinema.length; i++) {
+            for (int j = 0; j < cinema[i].length; j++) {
+                System.out.print(cinema[i][j] + "  ");
+            }
+            System.out.println();
+        }
+    }
+
+    public boolean isTicket(int row, int column){
+        int[][]arr = getCinema();
+        return (row - 1 >= 1 && row - 1 <= arr.length) && (column - 1 >= 1 &&  column - 1 <= arr[0].length);
+    }
+
+    public boolean checkTicket(int row, int column){
```

Comment:
jak zmienisz koncepcję zgodnie z moimi wskazówkami to ta metoda nie będzie potrzebna